### DISCUSSION TITLE
Documentation doesn't explain how to run tests

### DISCUSSION THREAD

**Discussion by yurivict** [BrunoLevy/geogram#187] [Q&A] [answered]
The documentation section [Geogram compilation on Linux](https://github.com/BrunoLevy/geogram/wiki/compiling_Linux) doesn't explain how to run tests.

**Comment by BrunoLevy** [comment, marked as answer]
How to run tests is explained [here](https://github.com/BrunoLevy/geogram/wiki/DevOps)
